modules/ossm-tutorial-bookinfo-install.adoc

```diff
@@ -21,11 +21,16 @@ This tutorial walks you through how to create a sample application by creating a
 The Bookinfo sample application cannot be installed on IBM Z and IBM Power Systems.
 ====
 
+[NOTE]
+====
+The commands in this section assume the control plane project is `istio-system`.  If you installed the control plane in another namespace, edit each command before you run it.
```

Comment:
I don't see this note on the page. Is it maybe on another?

Reply:
This module is in a different assembly, the preview is here -> https://deploy-preview-39931--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh.html#ossm-tutorial-bookinfo-overview_ossm-create-mesh